bundles/org.openhab.binding.groupepsa/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,559 @@
+				<option value="Locked">Locked</option>
+				<option value="SuperLocked">Super Locked</option>
+				<option value="DriverDoorUnlocked">Driver Door Unlocked</option>
+				<option value="CabinDoorsUnlocked">Cabin Doors Unlocked</option>
+				<option value="CargoDoorsLocked">Cargo Doors Locked</option>
+				<option value="CargoDoorsUnlocked">Cargo Doors Unlocked</option>
+				<option value="RearDoorsLocked">Rear Doors Locked</option>
+				<option value="RearDoorsUnlocked">Rear Doors Unlocked</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="doorOpen">
+		<item-type>Contact</item-type>
+		<label>Door open</label>
+		<description>Door is open</description>
+		<category>Door</category>
+	</channel-type>
+
+	<channel-type id="ignition">
+		<item-type>String</item-type>
+		<label>Ignition</label>
+		<description>Ignition state</description>
+		<category>Motion</category>
+		<state readOnly="true">
+			<options>
+				<option value="Stop">Stop</option>
+				<option value="StartUp">Start up</option>
+				<option value="Start">Start</option>
+				<option value="Free">Free</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="moving">
+		<item-type>Contact</item-type>
+		<label>Moving</label>
+		<description>Vehicle is moving</description>
+		<category>Motion</category>
+	</channel-type>
+
+	<channel-type id="acceleration">
+		<item-type>Number:Acceleration</item-type>
+		<label>Acceleration</label>
+		<description>Current acceleration</description>
+		<category>Motion</category>
+		<state pattern="%.1f %unit%"/>
+	</channel-type>
+
+	<channel-type id="speed">
+		<item-type>Number:Speed</item-type>
+		<label>Speed</label>
+		<description>Current speed</description>
+		<category>Motion</category>
+		<state pattern="%.1f %unit%"/>
+	</channel-type>
+
+	<channel-type id="mileage">
+		<item-type>Number:Length</item-type>
+		<label>Mileage</label>
+		<description>Total travelled distance</description>
+		<category>Motion</category>
+		<state pattern="%.1f %unit%"/>
+	</channel-type>
+
+	<channel-type id="position">
+		<item-type>Location</item-type>
+		<label>Position</label>
+		<description>Last known position</description>
+		<category>MoveControl</category>
+		<state pattern="%2$s°N %3$s°E %1$sm"/>
+	</channel-type>
+
+	<channel-type id="heading">
+		<item-type>Number:Angle</item-type>
+		<label>Heading</label>
+		<description>Direction of travel</description>
+		<category>MoveControl</category>
+		<state pattern="%.0f %unit%"/>
+	</channel-type>
+
+	<channel-type id="type">
+		<item-type>String</item-type>
+		<label>Position Type</label>
+		<description>Position acquisition type</description>
+		<category>MoveControl</category>
+		<state readOnly="true">
+			<options>
+				<option value="Estimate">Estimate</option>
+				<option value="Acquire">Acquire</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="signal">
+		<item-type>Number:Dimensionless</item-type>
```

Comment:
There's a predefined signal strength channel you could use. See https://www.openhab.org/docs/developer/bindings/thing-xml.html#system-state-channel-types